scripts/seed.py

```
@@ -0,0 +1,93 @@
+            "nickname": user["nickname"],
+            "password": hash_password(user["password"]),
+            "profile_img": user["profile_img"],
+            "created_at": datetime.now(UTC).isoformat(),
+        })
+    return users
+
+
+def seed():
+    """모든 테스트 데이터 생성"""
+
+    def seed():
```

Ruff (0.14.13)

Missing return type annotation for private function `seed`

Add return type annotation: `None`

(ANN202)